lib/connection/connection.js

```diff
@@ -1,7 +1,7 @@
 /*
  * Copyright (c) 2015-2021 Snowflake Computing Inc. All rights reserved.
  */
-const uuidv4 = require('uuid/v4');
+const { v4: uuidv4 } = require('uuid');
```

Comment:
why do we need this change in this PR? (also in other files there are the same changes)

Reply:
It is because of some conflict after adding nyc dependency

Reply:
I think we should update `uuid` in separate PR if it is really necessary

Reply:
But the tests fail without this change. I thin it was bug and now it is imported correctly: https://www.npmjs.com/package/uuid

Reply:
Current master test do not fail because of it - and nyc should not have impact on the production code

I'm not blocking with this comment, but it looks strange

Reply:
https://github.com/snowflakedb/snowflake-sdks-drivers-issues-teamwork/issues/515

Reply:
Uuid import was changed in another issue.